src/ArraySetops.chpl

```diff
@@ -0,0 +1,137 @@
+/* Array set operations
+ includes intersection, union, xor, and diff
+
+ currently, only performs operations with integer arrays 
+ */
+
+module ArraySetops
+{
+    use ServerConfig;
+
+    use BlockDist;
+    use SymArrayDmap;
+
+    use RadixSortLSD;
+    use Unique;
+    use Indexing;
+    use In1d;
+
+    use CommAggregation;
+
+    /*
+    Small bound const. Brute force in1d implementation recommended.
+    */
+    private config const sBound = 2**4; 
+
+    /*
+    Medium bound const. Per locale associative domain in1d implementation recommended.
+    */
+    private config const mBound = 2**25; 
+
+    // returns intersection of 2 arrays
+    proc intersect1d(a: [?aD] int, b: [aD] int, assume_unique: string) {
+      //if not unique, unique sort arrays then perform operation
+      if (assume_unique == "False") {
```

Comment:
I'm not sure what other places in the code do, but it seems like it'd be cleaner to pass in a `bool` here, and at the callsite check that assume_unique is either "true" or "false" and convert to the appropriate bool (just thinking about a case where the message is improperly formatted, but we're not checking for that.